README.md

```diff
@@ -31,24 +31,26 @@ When a major issue in "beta 3" is found on SOAK, we will follow the steps above
 Only pre-releases that are SOAKed for at least two weeks should be used as stable releases. After the soaking period was successful, follow these steps:
 
 1. fetch latest repo state: `git fetch`
-1. checkout the pre-release tag you want to make a stable release: e.g. `git checkout v1.1.0-rc.3`
-1. create new `stable-*` branch for that minor release: e.g. `git checkout -b stable-1.1.x`
-1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to your newly created `stable-*` branch (remember to also update revision)
+1. update kommanders `appVersion` to be stable by removing the pre-release suffix and commit that change to `master` (remember to also update revision)
 1. apply and push tags
    1. "base" semver based tag: e.g. `git tag v1.1.0 && git push origin v1.1.0`
    1. "consumable" stable tag(s) for supported k8s versions: e.g. `git tag stable-1.16-1.1.0 && git push origin stable-1.16-1.1.0`
+1. create new `stable-*` branch for that minor release: e.g. `git checkout -b stable-1.1.x`. that way future updates have an easy target.
```

Comment:
As mentioned in earlier comment, let's keep the previous version of the above steps that were removed, and thus remove this step.

Reply:
rather, I will repurpose this by adding the step to merge back stable branch :)

```suggestion
1. merge the `stable-*` branch back into master. there will be merge conflicts to resolve due to different versions of the latest revision file existing on both the master and stable branches. resolve the conflicts by:
  1. keeping the stable branch version of that revision file
  1. creating a new directory with the next patch version (e.g. `1.1.1`)
  1. moving master's version of that revision file and any others following it into that newly created directory
```
^ should new dir be next patch version or minor? I used patch since we separate dirs out by patch versions instead of minor (mentioned this in an earlier comment, kba uses `1.1.x` instead of `1.1.1`)

Reply:
The merge conflict resolution seems a little risky. Ideally I would like to see this out in a PR for more eyes to double check, my only concern is that hitting the merge button on the PR will also auto-delete the stable branch. We _can_ manually "un-delete" the branch but that also seems sketch and I'm sure people will forget to do it. Is there a GH exception we can make to the auto-delete rule to ignore branches prefixed with `stable-` for example?

Reply:
Yes we can mark the branch protected (it already is) and it will not be deleted 

<img width="896" alt="Screen Shot 2020-07-17 at 11 24 16 AM" src="https://user-images.githubusercontent.com/4225547/87818813-11b7fa80-c820-11ea-9c47-640f8cfaae02.png">

Reply:
see more detailed comment above.

tl;dr
we dont want to merge back a newly created `stable-*` to `master`, the two branches must be equal anyway. also, there must never be a conflict when trying to merge any `stable-*` branch back to `master`.

Reply:
> there will be merge conflicts to resolve due to different versions of the latest revision file existing on both the master and stable branches

Tbh I can't remember why there would be this merge conflict, we can reject this and revisit if in the next release we run into something like this again